[*] Running ...
[*] Dataset loaded: E:\Projects\CO\reevo-main/problems/mkp_aco\dataset/train100_dataset.npz with 5 instances.
D:\Anaconda3\envs\reevo\Lib\site-packages\numpy\core\fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
D:\Anaconda3\envs\reevo\Lib\site-packages\numpy\core\_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
Traceback (most recent call last):
  File "E:\Projects\CO\reevo-main/problems/mkp_aco/eval.py", line 58, in <module>
    obj = solve(prize, weight)
          ^^^^^^^^^^^^^^^^^^^^
  File "E:\Projects\CO\reevo-main/problems/mkp_aco/eval.py", line 23, in solve
    heu = heuristics(prize.copy(), weight.copy()) + 1e-9
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\Projects\CO\reevo-main\problems\mkp_aco\gpt.py", line 15, in heuristics_v2
    score[score < threshold] = 0
                                 
ValueError: operands could not be broadcast together with shapes (100,) (5,) 
